Add relevant demo material for Data Marketplace (#1136)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/cloud-scale-analytics/architectures/data-mesh-data-marketplace.md

In some cases, you might need an external data marketplace. External data marketplaces enable sharing of your data products with external partners. You can use [Azure Data Share](/azure/data-share) as a component.

> To learn more on data marketplace concept using Microsoft Purview, refer this demo https://youtu.be/7DUPcMwV-Ig which talks about how you can simplify data consumers journey using self-service data discovery and data access management capabilities.

Reviewer: ```suggestion
See [a demo of the data marketplace concept using Microsoft Purview](https://aka.ms/purviewselfservicedemo) for a discussion of how you can simplify data consumers journey using self-service data discovery and data access management capabilities.
```
